jxl/src/frame/modular.rs

```diff
@@ -440,6 +444,120 @@ impl FullModularImage {
     }
 }
 
+#[allow(dead_code)]
+#[allow(clippy::too_many_arguments)]
+fn dequant_lf<'a>(
+    r: Rect,
+    lf: (
```

Comment:
why not a [_; 3]?

Reply:
It seems to make it a pain to mutably borrow all three at the same time like on lines 481..483.

Reply:
You can `let [a, b, c] = array ` :-)

Reply:
Oh :D thanks.